pkg/gator/test/test.go

```diff
@@ -88,26 +94,52 @@ func Test(objs []*unstructured.Unstructured) (*types.Responses, error) {
 				obj.GroupVersionKind(), obj.GetNamespace(), obj.GetName(), err)
 		}
 
-		for targetName, r := range review.ByTarget {
-			targetResponse := responses.ByTarget[targetName]
-			if targetResponse == nil {
-				targetResponse = &types.Response{}
-				targetResponse.Target = targetName
+		// Attempt to expand obj and review resultant resources (if any)
+		allReviews := []*types.Responses{review}
+		baseNs := &corev1.Namespace{}
+		baseNs.SetName(obj.GetNamespace())
+		base := mutationtypes.Mutable{
+			Object:    obj,
```

Comment:
Should we mutate the object before expanding it, since this is a `gator` expansion?